src/separation-todo.md

```diff
@@ -0,0 +1,14 @@
+Script to update the src/codap folder in the dst-plugin:
+`rsync -av --existing --delete ../codap/v3/src/ src/codap/`
+To be complete after an update you need to review all of the files and see if the problem files list below needs to be updated.
+
+We are trying to avoid bringing in the document and top level tile infrastructure.
+TileContentModel and SharedModel are included to because they don't bring in many other dependencies and it was too hard to keep them out.
```

Comment:
```suggestion
TileContentModel and SharedModel are included because they don't bring in many other dependencies and it was too hard to keep them out.
```